R/rscpages.R

```diff
@@ -1,11 +1,12 @@
 #' rscpages: Curate your content on RStudio Connect
 #'
-#' rscpages provides UI components to help publishers provide curated organization of content on RStudio Connect.
+#' rscpages provides UI components to help publishers provide curated
+#' organization of content on RStudio Connect.
 #'
 #' @section Connection:
-#' Set `CONNECT_SERVER` and `CONNECT_API_KEY` environment variables and [connect()].
-#' Fetch a `tibble` listing server content with [content()].
-#' use the [by_owner()] and [by_tags()] filter helpers to narrow your search.
+#' Set `CONNECT_SERVER` and `CONNECT_API_KEY` environment variables and
+#' [connect()]. Fetch a `tibble` listing server content with [content()]. use
```

Comment:
Probably meant to be upper case on the next line?

Reply:
Fixed